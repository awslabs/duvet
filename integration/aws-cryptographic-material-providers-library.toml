name = "aws-cryptographic-material-providers-library"
repo = "https://github.com/aws/aws-cryptographic-material-providers-library"
version = "v1.7.0"
cmd = [
    "make duvet"
]
html_report = "specification_compliance_report.html"
