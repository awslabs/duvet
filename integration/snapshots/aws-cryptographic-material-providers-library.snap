version https://git-lfs.github.com/spec/v1
oid sha256:cc10932361d475761d40f151a66315aaec49e064805bbe9a3475cb17b1c8e103
size 1830803
